css-color-3/t424-hsl-values-b-1.xht

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<title>CSS Test: hsl() values</title>
```

Comment:
Since you have split the tests, please make the title more concise, if possible.